team/src/App.js

```diff
@@ -1,20 +1,81 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import { StreamChat } from 'stream-chat';
+import { Chat, enTranslations, Streami18n } from 'stream-chat-react';
+import { createGlobalStyle } from 'styled-components';
+import 'stream-chat-react/dist/css/index.css';
+
 import './App.css';
 
-function App() {
+import { ChannelContainer } from './components/ChannelContainer/ChannelContainer';
+import { ChannelListContainer } from './components/ChannelListContainer/ChannelListContainer';
+import { ColorSlider } from './components/ColorSlider/ColorSlider';
+
+const urlParams = new URLSearchParams(window.location.search);
+const apiKey = urlParams.get('apikey') || process.env.REACT_APP_STREAM_KEY;
+const user = urlParams.get('user') || process.env.REACT_APP_USER_ID;
+const theme = urlParams.get('theme') || 'light';
+const userToken =
+  urlParams.get('user_token') || process.env.REACT_APP_USER_TOKEN;
+
+const i18nInstance = new Streami18n({
+  language: 'en',
+  translationsForLanguage: {
+    ...enTranslations,
+    // '1 reply': '1 thread reply',
+    // '{{ replyCount }} replies': '{{ replyCount }} thread replies',
+    Flag: 'Pin to Channel',
```

Comment:
We should probably not use the flag function for pinning. Good to make a note to implement this properly. Not a priority for now

Reply:
Agreed! This was really the most feasible work around without having to create an entirely new component, just hijack the existing functionality and change it's name and action handler. We should be getting guidance from Ben and the backend team on pin functionality soon as well